.github/CODEOWNERS

```diff
@@ -1483,6 +1483,7 @@ x-pack/solutions/observability/plugins/observability/server/lib/esql_extensions
 /x-pack/solutions/observability/plugins/observability/public/pages/overview @elastic/obs-ux-infra_services-team # Assigned to this team since it mostly uses infra/APM components
 /src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces @elastic/obs-ux-infra_services-team
 /src/platform/plugins/shared/unified_doc_viewer/public/components/observability/attributes @elastic/obs-ux-infra_services-team
+/src/platform/packages/shared/kbn-unified-metrics-grid @elastic/obs-ux-infra_services-team
```

Comment:
I added this after I had the PR opened, but I still need to add our team to the reviewers list manually🤔 Maybe after merging it, it will work fine for the next PRs 🤞